docs/install-audit-log-filter.md

```diff
@@ -0,0 +1,42 @@
+# Install the Audit Log Filter
+
+The feature is in [tech preview](glossary.md#tech-preview).
+
+The `plugin_dir` system variable defines the plugin library location. If needed, at server startup, set the `plugin_dir` variable.
+
+When upgrading a MySQL installation, plugins are not automatically upgraded. You may need to manually load certain functions or reinstall the plugin after the MySQL upgrade.
+
+In the `share` directory, locate the `audit_log_filter_linux_install.sql `script.
+
+Prior to 8.0.34, use this command:
+
+```{.bash data-prompt="$"}
+$ mysql -u -p < audit_log_filter_linux_install.sql
+```
+
+Implemented in 8.0.34, at the time you run the script, you can select the database used to store the JSON filter tables. If you do use the `-D` option, the tables are installed in the default system database, `mysql`. You can also designate a different database with the `audit_log_filter_database` system variable. The database name cannot be NULL or exceed 64 characters. If the database name is invalid, the audit log filter tables are not found.
```

Comment:
"If you do use the `-D` option, the tables are installed in the default system database, `mysql`."

This works in a bit different way. Possible options here:
- In case plugin is loaded, installation script will take DB name from audit_log_filter_database variable.
- In case plugin is not yet loaded but "-D db_name" is passed to mysql client when running installation script, this db_name will be used.
- Otherwise DB name used by installation script defaults to 'mysql'.